fbgemm_gpu/include/fbgemm_gpu/embedding_backward_template_helpers.cuh

```diff
@@ -10,9 +10,17 @@
 #include <ATen/TensorUtils.h>
 #include <ATen/core/TensorAccessor.h>
 #include <ATen/cuda/CUDAContext.h>
+#if !defined(NEW_GENERATOR_PATH)
+#include <ATen/CUDAGeneratorImpl.h>
```

Comment:
Can we remove `NEW_GENERATOR_PATH` and only use `ATen/cuda/CUDAGeneratorImpl.h` for PyTorch 1.11+ ?

Reply:
We moved to ROCm5.1 and PyTorch1.11. These branches were deleted.